packages/transaction-controller/src/TransactionController.ts

```diff
@@ -663,7 +663,11 @@ export class TransactionController extends BaseController<
       providerConfig: { type: chain, chainId, nickname: name },
     } = this.getNetworkState();
 
-    if (chain !== RPC) {
+    if (
```

Comment:
I know there aren't any tests for `getCommonConfiguration`, but what do you think about adding a test for this behavior?